aep/general/0137/aep.md.j2

```diff
@@ -0,0 +1,143 @@
+# Apply
+
+In REST APIs, it is customary to make a `PUT` request to a collection's URI (for
+example, `/v1/publishers/{publisher}/books`) in order to create or update a new
+resource within that collection.
+
+Resource-oriented design (AEP-121) honors this pattern through the `Apply`
```

Comment:
How will `Resource-oriented design (AEP-121)` get rendered - will it have a lozenge with the AEP's name?  I noticed that below you have a reference to an AEP without a preceding title...

Reply:
@earth2marsh suggested the lozenges, and here's an issue to make sure they render similar to the old site: https://github.com/aep-dev/site-generator/issues/52

Reply:
I'll resolve this one since it's out of scope for the PR, but feel free to re-open if you want me to do something explicit in this one.